### DISCUSSION TITLE
How to customise horizontal matrix/array element spacing

### DISCUSSION THREAD

**Discussion by Lordfirespeed** [KaTeX/KaTeX#3868] [Q&A] [answered]
I'm struggling to find a way to customise the horizontal 'gutter' spacing width in-between elements in a `matrix` or `array` environment - Is that functionality supported? Could I see an example, if it is?

**Comment by edemaine** [comment, marked as answer]
I assume you mean `arraycolsep`, which is not yet supported; see #687.

**Reply by Lordfirespeed** [reply]
Thankyou!
